src/pybind/mgr/dashboard/module.py

```diff
@@ -250,7 +250,7 @@ class Module(MgrModule, CherryPyConfig):
         {'name': 'crt_file'},
         {'name': 'ssl'}
     ]
-    OPTIONS.extend(options_schema_list())
+    MODULE_OPTIONS.extend(options_schema_list())
```

Comment:
Should `options_schema_list()` also be renamed to `module_options_schema_list()`?

Reply:
it's not part of the mgr/module interface so i'm inclined to leave it for the dashboard folks?

Reply:
Fine by me